.github/workflows/build_and_test.yml

```diff
@@ -339,17 +339,17 @@ jobs:
         run: |
           ${cov_prefix} mold -run cargo build $CARGO_FLAGS $CARGO_FEATURES --bins --tests
 
-      - name: Run cargo test
+      - name: Run nextest test
         run: |
-          ${cov_prefix} cargo test $CARGO_FLAGS $CARGO_FEATURES
+          ${cov_prefix} cargo nextest run $CARGO_FLAGS $CARGO_FEATURES
 
           # Run separate tests for real S3
           export ENABLE_REAL_S3_REMOTE_STORAGE=nonempty
-          export REMOTE_STORAGE_S3_BUCKET=neon-github-public-dev
+          export REMOTE_STORAGE_S3_BUCKET=neon-github-ci-tests
```

Comment:
We have these changes in the main, and they shouldn't be in the PR.

Reply:
I think there is something wrong with my rebase. I am currently fixing it. Hopefully it should not show up with my next push

Reply:
after rebasing with main this has not disappeared